dark/index.html

```diff
@@ -194,21 +194,6 @@ <h1>CONTACT ME</h1>
       <h3>CONTACT <span class="text-info">ME</span></h3>
 
       <div class="formContainer bg-dark p-5 rounded-3">

```

Comment:
Remove this line also and the related css codes @PiyushXCoder

Reply:
I have pushed the changes. This is what it looks like.

![Uploading Screenshot 2024-10-08 at 16-58-41 YOSHITHA RATHNAYAKE.png…]()